src/components/YourCompany/YourCompany.vue

```diff
@@ -226,21 +226,17 @@
     <!-- Registered Office Contact Information -->
     <div id="contact-info" class="section-container" :class="{'invalid-section': invalidContactSection}">
       <BusinessContactInfo
-        @haveChanges="contactInfoChanges = $event"
+        :invalidSection="invalidContactSection"
       />
     </div>
 
     <!-- Folio Information -->
-    <template v-if="isPremiumAccount">
+    <template v-if="isPremiumAccount && !isRoleStaff">
```

Comment:
This means FN doesn't display in correction (any more) but I think that's OK since it can't be changed in correction. I have asked Linda and will update this as needed.
Also, the comps say to show the FN but hide the Change button for staff, which isn't what I implemented. I have asked Linda about this as well.
Either way, I will change this later (not in this PR).

Reply:
Sounds good!

Reply:
Linda asked that I fix this (in future PR)... so for both corrections and alterations, the FN will display but cannot be changed.